syntax = "proto2";

package FactRouteDto.proto2;

message FactRouteDto {
  optional string barRecordId = 1;
  optional string waybillNo = 2;
  optional string zoneCode = 3;
  optional string opCode = 4;
  optional string opAttachInfo = 5;
  optional string courierCode = 6;
  optional string barOprCode = 7;
  optional fixed64 barUploadTm = 8;
  optional string objTypeCode = 9;
  optional string contnrCode = 10;
  optional int64 payFlg = 11;
  optional string stayWhyCode = 12;
  optional int64 subbillPieceQty = 13;
  optional int64 barUploadTypeCode = 14;
  optional double weightQty = 15;
  optional double feeAmt = 16;
  optional string accountantCode = 17;
  optional string otherInfo = 18;
  optional string opName = 19;
  optional int64 zoneTypeCode = 20;
  optional string encryptString = 21;
  optional string barSn = 22;
  optional string scheduleCode = 23;
  optional string signTypeCode = 24;
  optional string srcContnrCode = 25;
  optional string phoneZone = 26;
  optional string phone = 27;
  optional int64 stopOverFlg = 28;
  optional string batchCode = 29;
  optional string destZoneCode = 30;
  optional string autoloading = 31;
  optional fixed64 barScanTm = 32;
  optional fixed64 barScanDt = 33;
  optional string barUploadOprCode = 34;
  optional string extendAttach1 = 35;
  optional string extendAttach2 = 36;
  optional string extendAttach3 = 37;
  optional string extendAttach4 = 38;
  optional string extendAttach5 = 39;
  optional string extendAttach6 = 40;
  optional string extendAttach7 = 41;
  optional fixed64 barUploadTmStd = 42;
  optional fixed64 barScanTmStd = 43;
  optional int32 source = 44;
//  optional MapStringString exts = 45;
  map <string, string> exts = 45;
  optional int32 dataStatus = 50;
  optional int32 status = 51;
  optional string mainWaybillNo = 52;
  optional int64 ordinal = 53;
  optional fixed64 createTm = 54;
}